quickstart.md

```diff
@@ -149,6 +149,7 @@ Details on each of the runtime classes:
 - *kata-qemu* - same as kata
 - *kata-qemu-tdx* - using QEMU, with TDVF, and support for Intel TDX CC HW, prepared for using Verdictd and EAA KBC.
 - *kata-qemu-sev* - using QEMU, and support for AMD SEV HW
+- *kata-qemu-snp* - using QEMU, and support for AMD SNP HW
```

Comment:
These runtime class names aren't up-to-date with the latest operator. If you can install the operator, you should get the new set of runtime classes and update them.

Reply:
Updated